### DISCUSSION TITLE
Sprites location is incorrect

### DISCUSSION THREAD

**Discussion by Gonzalo-Bruna** [ppy/osu#17686] [Q&A]
after dragging and dropping a new sprite, the position where it ends is incorrect. It might be because I just grabbed the first picture of my dog I found and it's quite big in size.

(video starts at 7 seconds)

https://user-images.githubusercontent.com/41755816/162018776-c0719834-8424-483c-88b9-78a715f47c2a.mp4

here's the image I used:
https://drive.google.com/file/d/1TNfQS_nUnB5y5zLX1oh0m179_H9zDh2o/view?usp=sharing

**Comment by bdach** [comment]
the positioning may not be what you're looking for but it isn't incorrect. the sprite is positioned relative to the bottom right corner (see the long yellow line pointing to said corner). if you set the anchor to top left, you'll probably get behaviour closer to expectation.

that said, this is also partially due to aspect ratio changing between preview and no preview, which is under scope of https://github.com/ppy/osu/discussions/17407.

**Reply by peppy** [reply]
The weird part is, by default the anchor should be automatically updating when dragging the sprite into a new location. The fact that it isn't means the user likely set it incorrectly, right?

https://user-images.githubusercontent.com/191335/162122639-ec201f36-1ea1-4f72-944a-a6ad52385a61.mp4
